bundle/android/strings.xml

```diff
@@ -194,7 +197,6 @@
     <string name="gutenberg_native_get_support" tools:ignore="UnusedResources">Get support</string>
     <string name="gutenberg_native_gif" tools:ignore="UnusedResources">GIF</string>
     <string name="gutenberg_native_give_it_a_try_by_adding_a_few_blocks_to_your_post_or_page" tools:ignore="UnusedResources">Give it a try by adding a few blocks to your post or page!</string>
-    <string name="gutenberg_native_go_back" tools:ignore="UnusedResources">Go back</string>
```

Comment:
This string is still used but since it's now also in the web version, it will be translated via Gutenberg's GlotPress project ([reference](https://github.com/WordPress/gutenberg/blob/980cad0d37b827dab245797c8d67cf90d0505256/packages/editor/src/components/post-template/block-theme.js#L72)).

Reply:
Ohhh TIL! Thank you!